ush/forecast_postdet.sh

```diff
@@ -663,14 +667,17 @@ WW3_nml() {
 }
 
 WW3_out() {
+  mkdir -p "${COM_CONF}"
```

Comment:
Shouldn't be necessary if this is done in `common_predet`

Reply:
Done.